src/Highlighter.ts

```diff
@@ -145,7 +146,11 @@ export default class Highlighter {
         .filter((other: Highlight) => other.intersects(range));
 
       if (highlights.length === 0) {
-        const highlight: Highlight = new Highlight(range, {content: rangeContentsString(range)});
+        const highlight: Highlight = new Highlight(
+          range,
+          { content: rangeContentsString(range) },
+          { skipIDsBy: this.options.skipIDsBy }
```

Comment:
can we create a new method `Highlighter.getHighlightOptions` which will return the subset of `options` that should be passed to the highlight constructor? after that can you also use it in `SerializedHighlight.load` when constructing highlights there? that way all highlights will have the same options regardless of if they were loaded from serialized data or made through the UI, and loading a highlight and serializing it again won't change the reference element

Reply:
can use getHighlightOptions here